applications/StructuralMechanicsApplication/custom_utilities/tangent_operator_calculator_utility.h

```diff
@@ -157,6 +230,34 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) TangentOperatorCalculatorUtil
         rPerturbation = std::max(perturbation_1, perturbation_2);
     }
```

Comment:
Not necessarily, anyway the code is not in green, which means it was already like that, can you contribute @AlejandroCornejo ?

Reply:
it is not clear because we do not know if the pert_1 is the minimum because it is multiplied by 1e-5 whereas the maximum component is multiplied b y 1e-10.

perturbation_1 = PerturbationCoefficient1 * min_strain_component;
perturbation_2 = PerturbationCoefficient2 * max_strain_component;

Reply:
ok. We can check later if we can improve this checking...